packages/gallery/src/components/gallery/blueprintsIndex.js

```diff
@@ -16,8 +16,22 @@ export default class BaseGallery extends React.Component {
     let galleryProps = { ...otherProps, styles: _styles, eventsListener: _eventsListener, domId};
 
 
-    if(this.props.useBlueprints) {
-      //
+    if (this.props.useBlueprints) {
+      try {
+        const {layoutFixer} = window;
+        if (typeof window !== 'undefined' && layoutFixer && !layoutFixer.hydrated) {
+          galleryProps.structure = layoutFixer.structure;
+          galleryProps.container = layoutFixer.container;
+          galleryProps.items = layoutFixer.items;
+          layoutFixer.hydrated = true;
+          console.log('[LAYOUT FIXER] used structure and container from layoutFixer', layoutFixer);
+        } else {
+          return null;
```

Comment:
This will block any BP gallery that doesn't use layoutFixer (although it is a feature, not a must), even after wrapper measurements are 100% good.